doc/articles/common-issues-ioscatalyst.md

```diff
@@ -19,3 +19,13 @@ In order to fix this, add the following to your csproj (Xamarin, `net6.0-ios`, `
   <MtouchExtraArgs>$(MtouchExtraArgs) --registrar=static</MtouchExtraArgs>
 </PropertyGroup>
 ```
+## Error while retreving IOS device in VS code
+
+When switching to an iOS debugging target in VS Code, you might encounter an error stating that the iOS device could not be retrieved. The error message may appear as follows:
+
+```
+[Info]: Project reload forced to switch to net8.0-ios | Debug
+[Error] Could not retrieve ios devices within 10 seconds. Aborting...
+```
+To resolve this issue, you can download Xcodes. Open the latest version of Xcode and click the Make Active button. After completing this step, restart your machine. This should resolve the error when switching to an iOS debugging target in VS Code.
```

Comment:
you should add a link to https://www.xcodes.app since it can be confusing this is not just download Xcode (with an extra `s`)

> Open the latest version of Xcode and click the Make Active

That can be confusing too. Maybe something like:

`Inside Xcodes.app, select the correct version of Xcode and click the **Make Active** button to make it the default Xcode for your Mac.`

> After completing this step, restart your machine.

No, there's no need for such drastic measure. This is not Windows ;-)

You should ask VS Code to `Developer: Reload Window` to speed things up. This will use the new (default) Xcode to ask for the simulators.